README.md

```diff
@@ -0,0 +1,116 @@
+# **ALTERNATE BIT PROTOCOL USING CADMIUM**
+---
```

Comment:
One of the suggestion (not mandatory) would be to include table of contents and provide hyperlinks to the respective section. This will allow users to jump to navigate between sections easily and provides better visibility to the file.

Reply:
Thanks for suggestion i have implemented that also